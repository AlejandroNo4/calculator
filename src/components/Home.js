import React from 'react';

const Home = () => (
  <div>
    <div className="home-container">
      <h1 className="title">Welcome to our page!</h1>
      <p className="text">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla efficitur
        imperdiet velit quis mollis. Donec eget ante nec odio consectetur
        interdum ac vel neque. Cras sed commodo mi. Aliquam erat volutpat. Morbi
        efficitur ex odio. Lorem ipsum dolor sit amet, consectetur adipiscing
        elit. Donec ut lacus dapibus, suscipit libero vitae, egestas justo.
      </p>
      <p className="text">
        Aenean id velit nisi. Suspendisse potenti. Nunc pulvinar justo nisl, vel
        maximus mauris egestas consectetur. Nulla eget tortor at tellus placerat
        gravida at at tellus. Sed porttitor, justo ac pretium scelerisque, velit
        sem gravida urna, eget blandit risus est ac eros. Suspendisse eu urna id
        nulla convallis molestie. Curabitur massa mauris, porttitor ac
        vestibulum non, rhoncus in orci. Vestibulum ante ipsum primis in
        faucibus orci luctus et ultrices posuere cubilia curae; Cras faucibus
        nunc sit amet sapien sollicitudin, sed faucibus justo ultricies. Donec
        ut felis eu orci viverra tempor. Nullam neque lectus, eleifend eget
        augue eget, commodo tincidunt erat. Vivamus nec convallis lectus. Sed
        congue nisi id luctus auctor. Duis feugiat eros rutrum, pretium lectus
        ut, gravida nunc. Fusce hendrerit erat cursus massa iaculis consectetur.
      </p>
    </div>
  </div>
);

export default Home;
