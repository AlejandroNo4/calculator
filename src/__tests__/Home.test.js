import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Home from '../components/Home';

test('render component', () => {
  const component = render(<Home />);
  
});
