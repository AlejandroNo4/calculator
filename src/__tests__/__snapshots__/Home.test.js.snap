// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="home-container"
        >
          <h1
            class="title"
          >
            Welcome to our page!
          </h1>
          <p
            class="text"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla efficitur imperdiet velit quis mollis. Donec eget ante nec odio consectetur interdum ac vel neque. Cras sed commodo mi. Aliquam erat volutpat. Morbi efficitur ex odio. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ut lacus dapibus, suscipit libero vitae, egestas justo.
          </p>
          <p
            class="text"
          >
            Aenean id velit nisi. Suspendisse potenti. Nunc pulvinar justo nisl, vel maximus mauris egestas consectetur. Nulla eget tortor at tellus placerat gravida at at tellus. Sed porttitor, justo ac pretium scelerisque, velit sem gravida urna, eget blandit risus est ac eros. Suspendisse eu urna id nulla convallis molestie. Curabitur massa mauris, porttitor ac vestibulum non, rhoncus in orci. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras faucibus nunc sit amet sapien sollicitudin, sed faucibus justo ultricies. Donec ut felis eu orci viverra tempor. Nullam neque lectus, eleifend eget augue eget, commodo tincidunt erat. Vivamus nec convallis lectus. Sed congue nisi id luctus auctor. Duis feugiat eros rutrum, pretium lectus ut, gravida nunc. Fusce hendrerit erat cursus massa iaculis consectetur.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="home-container"
      >
        <h1
          class="title"
        >
          Welcome to our page!
        </h1>
        <p
          class="text"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla efficitur imperdiet velit quis mollis. Donec eget ante nec odio consectetur interdum ac vel neque. Cras sed commodo mi. Aliquam erat volutpat. Morbi efficitur ex odio. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ut lacus dapibus, suscipit libero vitae, egestas justo.
        </p>
        <p
          class="text"
        >
          Aenean id velit nisi. Suspendisse potenti. Nunc pulvinar justo nisl, vel maximus mauris egestas consectetur. Nulla eget tortor at tellus placerat gravida at at tellus. Sed porttitor, justo ac pretium scelerisque, velit sem gravida urna, eget blandit risus est ac eros. Suspendisse eu urna id nulla convallis molestie. Curabitur massa mauris, porttitor ac vestibulum non, rhoncus in orci. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras faucibus nunc sit amet sapien sollicitudin, sed faucibus justo ultricies. Donec ut felis eu orci viverra tempor. Nullam neque lectus, eleifend eget augue eget, commodo tincidunt erat. Vivamus nec convallis lectus. Sed congue nisi id luctus auctor. Duis feugiat eros rutrum, pretium lectus ut, gravida nunc. Fusce hendrerit erat cursus massa iaculis consectetur.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
